packages/experience-builder-sdk/src/core/editor/designComponentUtils.ts

```diff
@@ -33,10 +37,18 @@ export const deserializeDesignComponentNode = ({
     if (variable.type === 'BoundValue') {
       const [, uuid, ,] = variable.path.split('/');
 
-      dataSource[uuid] = { ...experienceDataSource[uuid] };
+      dataSource[uuid] = { ...designComponentDataSource[uuid] };
     } else if (variable.type === 'UnboundValue') {
```

Comment:
Is there a scenario where the variable type would be Bound value or unbound value? I thought we'd only have `ComponentValue` and `DesignValue` as variables types, and on looking up the `variableMapping`, we know if it's a bound or unbound value

Reply:
There shouldn't be a scenario, I can remove this old code

Reply:
Maybe later on, when we allow to fix content variables within the design component, they might be direct unbound values / data source instead of being configurable through a ComponentValue but that's more an idea that we might implement or not.